validator/slashing-protection/local/standard-protection-format/import.go

```diff
@@ -267,18 +259,39 @@ func filterSlashablePubKeysFromAttestations(
 	signedAttsByPubKey map[[48]byte][]*kv.AttestationRecord,
 ) ([][48]byte, error) {
 	slashablePubKeys := make([][48]byte, 0)
+	// First we need to find attestations that are slashable with respect to other
+	// attestations within the same JSON import.
 	for pubKey, signedAtts := range signedAttsByPubKey {
+		signingRootsByTarget := make(map[uint64][32]byte)
+		targetEpochsBySource := make(map[uint64][]uint64)
+	Loop:
```

Comment:
These changes allow us to check if an attestation in the imported JSON is slashable with respect to other attestations in the same JSON